lib/create-contentful-api.js

```diff
@@ -141,10 +141,15 @@ export default function createClientApi ({ http }) {
     .then((response) => wrapOrganizationCollection(http, response.data), errorHandler)
   }
 
+  function rawRequest (opts) {
```

Comment:
one more thing, can you add the function doc 🤓 . You can copy paste from other functions and change accordingly